cwatch.php

```diff
@@ -1159,4 +1302,22 @@ private function getServiceRules(array $vars = null, $edit = false)
 
         return $rules;
     }
+
+    /**
+     * Validates that the given hostname is valid
+     *
+     * @param string $host_name The host name to validate
+     * @return bool True if the hostname is valid, false otherwise
+     */
+    public function validateHostName($host_name)
```

Comment:
Does this method need to exist or can it be an anonymous function for the rules?